ddp-workspace/projects/ddp-sdk/src/lib/services/submitAnnouncement.service.ts

```diff
@@ -14,18 +13,13 @@ import { ActivityResponse } from '../models/activity/activityResponse';
  */
 @Injectable()
 export class SubmitAnnouncementService {
-    private _submitAnnounced$: Observable<ActivityResponse>;
-    private submitAnnouncedSource: Subject<ActivityResponse>;
+    private submitAnnouncedSource = new Subject<boolean>();
 
-    constructor() {
-        this.submitAnnouncedSource = new Subject<ActivityResponse>();
-        this._submitAnnounced$ = this.submitAnnouncedSource.asObservable();
-    }
-    get submitAnnounced$(): Observable<ActivityResponse> {
-      return this._submitAnnounced$;
+    get submitAnnounced$(): Observable<boolean> {
+      return this.submitAnnouncedSource.asObservable();
     }
 
-    public announceSubmit(response: ActivityResponse): void {
-        this.submitAnnouncedSource.next(response);
+    public announceSubmit(submitOccurred: boolean): void {
+        this.submitAnnouncedSource.next(submitOccurred);
     }
```

Comment:
We never used it for passing response, only for emitting a signal that submit is fired.
(see also https://github.com/broadinstitute/ddp-angular/pull/735/files#r676509192)